.gitignore

```diff
@@ -1,27 +1,27 @@
-/node_modules/
-
-coverage
-#Compiler Output
-*.tsbuildinfo
-*.log
-lib
-
-test/**/*.js
-test/**/*.js.map
-test/**/*.d.ts
-test/**/*.d.ts.map
-
-src/**/*.js
-src/**/*.js.map
-src/**/*.d.ts
-src/**/*.d.ts.map
-
-lit-mobx.js
-lit-mobx.js.map
-lit-mobx.d.ts
-lit-mobx.d.ts.map
-
-demo/**/*.js
-demo/**/*.js.map
-
-.DS_Store
+/node_modules/
+
+/coverage
+#Compiler Output
+*.tsbuildinfo
+*.log
+/lib
```

Comment:
this subtle change stopped git ignorning everything inside the `src/lib` folder.